pkg/kubelet/server/server.go

```diff
@@ -417,6 +419,21 @@ func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
 	}
 }
 
+// InstallDebuggingDisabledHandlers registers the HTTP request patterns that provide better error message
+func (s *Server) InstallDebuggingDisabledHandlers() {
+
+	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		http.Error(w, "Debug endpoints are disabled.", http.StatusMethodNotAllowed)
+	})
+
+	paths := []string{
+		"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/",
+		"/runningpods/", "/pprofBasePath/", "/logsPath/"}
```

Comment:
logsPath & pprofBasePath are variables. It should be:
```go
"/runningpods/", pprofBasePath, logsPath}
```

Reply:
o... really sorry for this.